### PR TITLE
Update meeting-policies-in-teams.md

### PR DESCRIPTION
**Pull request by rarang**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/meeting-policies-in-teams.md [hunk 1]
```diff
@@ -549,17 +549,19 @@ This is a per-user policy. This setting controls whether users can customize the
 
 Currently, you can only use PowerShell to set this policy. You can edit an existing Teams meeting policy by using the [Set-CsTeamsMeetingPolicy](/powershell/module/skype/set-csteamsmeetingpolicy) cmdlet. Or, create a new Teams meeting policy by using the [New-CsTeamsMeetingPolicy](/powershell/module/skype/new-csteamsmeetingpolicy) cmdlet, and then assign the policy to users.
 
-To specify whether users can customize their video background in a meeting, set the **VideoFiltersMode** parameter as follows:
+To specify whether users can customize their video background in a meeting or consume scenes outside of the company and Microsoft Teams Store, set the **VideoFiltersMode** parameter as follows:
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-in-teams.md:552]:
```suggestion
To specify whether users can customize their video background in a meeting or use scenes from outside the organization and Microsoft Teams Store, set the **VideoFiltersMode** parameter as follows:
```

File: Teams/meeting-policies-in-teams.md [hunk 2]
```diff
@@ -549,17 +549,19 @@ This is a per-user policy. This setting controls whether users can customize the
...
 
 |Setting value in PowerShell |Behavior  |
 |---------|---------|
 |**NoFilters**     |User can't customize their video background.|
 |**BlurOnly**     |Users can blur their video background. |
 |**BlurandDefaultBackgrounds**     |User has the option to blur their video background or choose from the default set of images to use as their background. |
-|**AllFilters**     |User has the option to blur their video background, choose from the default set of images, or upload custom images to use as their background. |
+|**AllFilters**     |User has the option to blur their video background, choose from the default set of images, or upload custom images to use as their background, and consume Together Mode scenes outside of the Microsoft Teams Store and Company Apps Store (LoB). |
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-in-teams.md:559]:
```suggestion
|**AllFilters**     |User has the option to blur their video background, choose from the default set of images, or upload custom images to use as their background, and use Together Mode scenes from outside the Microsoft Teams Store and Company Apps Store. |
```

File: Teams/meeting-policies-in-teams.md [hunk 3]
```diff
@@ -549,17 +549,19 @@ This is a per-user policy. This setting controls whether users can customize the
...
 
 > [!IMPORTANT]
 > Images uploaded by users aren't screened by Teams. When you use the **AllFilters** setting, you should have internal organization policies to prevent users from uploading offensive or inappropriate images, or images your organization don't have rights to use for Teams meeting backgrounds.
+> For Together Mode Scenes, you are responsible for respecting others' rights, including copyright.
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-in-teams.md:563]:
```suggestion
> For Together Mode scenes, you are responsible for respecting others' rights, including copyright.
```

File: Teams/meeting-policies-in-teams.md [hunk 4]
```diff
@@ -549,17 +549,19 @@ This is a per-user policy. This setting controls whether users can customize the
...
+> Together Mode scenes available through Store can be governed through regular Apps Permission policies. Learn more about [Together Mode](https://review.docs.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/teams-together-mode?branch=pr-en-us-2710#create-an-app-package-and-use-the-scene-in-teams) and [Policies](https://docs.microsoft.com/en-us/microsoftteams/teams-app-permission-policies) by reviewing the links
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-in-teams.md:564]:
```suggestion
> Together Mode scenes available through Store can be governed through regular Apps Permission policies. To learn more about Apps Permission policies, see [](url) (https://docs.microsoft.com/en-us/microsoftteams/teams-app-permission-policies) by reviewing the links
```

Comment (JohanFreelancer9) [reply Teams/meeting-policies-in-teams.md:564]: The Together Mode link does not seem to work.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @rarang, please see my suggested changes and if you agree, please commit them.

**Comment by rarang** [comment]
@JohanFreelancer9 when will this get published

**Comment by JohanFreelancer9** [comment]
@rarang - You seem to have closed the PR. Was that intentional? If not, please reopen and commit all my suggested changes to move forward. (Click on **Commit** in each case.) Thanks.

File: Teams/meeting-policies-in-teams.md (hunk 4 above)
Comment (rarang) [reply Teams/meeting-policies-in-teams.md:564]: Can you word it this way - "Together Mode scenes available through the Teams Store can be governed through regular app permission policies"

**Review by rarang — COMMENTED** [review]
Can you please make the one change I requested

**Comment by rarang** [comment]
@JohanFreelancer9 I opened it and have committed all the changes, except for one small change. Please let me know

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@rarang - I have reworded my suggestion as you requested. Could you please commit this suggested change as well? Many thanks.

**Review by rarang — COMMENTED** [review]
This is good to go

**Comment by rarang** [comment]
@JohanFreelancer9 let's merge this

File: Teams/meeting-policies-in-teams.md [hunk 5]
```diff
@@ -549,17 +549,19 @@ This is a per-user policy. This setting controls whether users can customize the
...
+|**AllFilters**     |User has the option to blur their video background, choose from the default set of images, or upload custom images to use as their background, and use Together Mode scenes from outside the Microsoft Teams Store and Company Apps Store. |
 
 > [!IMPORTANT]
 > Images uploaded by users aren't screened by Teams. When you use the **AllFilters** setting, you should have internal organization policies to prevent users from uploading offensive or inappropriate images, or images your organization don't have rights to use for Teams meeting backgrounds.
+> For Together Mode scenes, you are responsible for respecting the rights of others, including copyright.
+> Together Mode scenes available through Store can be governed through regular Apps Permission policies. Learn more about [Together Mode](https://review.docs.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/teams-together-mode?branch=pr-en-us-2710#create-an-app-package-and-use-the-scene-in-teams) and [Policies](https://docs.microsoft.com/en-us/microsoftteams/teams-app-permission-policies) by reviewing the links
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-in-teams.md:564]:
```suggestion
> Together Mode scenes available through the Teams Store can be governed through regular app permission policies. Learn more about [Together Mode](/microsoftteams/platform/apps-in-teams-meetings/teams-together-mode?branch=pr-en-us-2710#create-an-app-package-and-use-the-scene-in-teams) and [Policies](/microsoftteams/teams-app-permission-policies) by reviewing the links.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@rarang - please take a look and commit my suggested changes. The links need to be relative links.

**Comment by rarang** [comment]
@JohanFreelancer9 I just did

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by JohanFreelancer9** [comment]
@scanum - Copy review complete.

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tiburd** [comment]
Closing PR because of age and inactivity.

cc @scanum

**Pull request closed without merging**